Graphs/ChordGraph/index.js

```diff
@@ -63,7 +57,7 @@ export default class ChordGraph extends AbstractGraph {
                 } = this.hoveredDatum;
 
                 return (
-                    <div>
+                    <React.Fragment>
                         <div>
```

Comment:
do we need the second div here?

Reply:
Yes, we need this div for the styling purpose.